src/commerce.js

```diff
@@ -118,6 +118,9 @@ class Commerce {
     const { eventCallback } = this.options;
     return promise.then(response => {
       if (response.status >= 200 && response.status < 300) {
+        if (response.data.constructor === 'Array') {
```

Comment:
Can we just check that it's not an object?
```suggestion
        if (typeof response.data !== 'object' || Array.isArray(response.data)) {
```